content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -126,7 +126,16 @@ command. In that case, you should explicitly set `--certificate-renewal=true`.
 
 You can renew your certificates manually at any time with the `kubeadm certs renew` command.
 
-This command performs the renewal using CA (or front-proxy-CA) certificate and key stored in `/etc/kubernetes/pki`.
+This command performs the renewal using CA (or front-proxy-CA) certificate and key stored in `/etc/kubernetes/pki`. 
+
+After running the command you should restart control-plane Pods because they do not support dynamic 
+certificate reload. You can not use `kubectl` because these are 
+[static Pods](/docs/tasks/configure-pod-containerstatic-pod/) and are not managed by the API Server. 
+So to restart any static Pod you can temporarily remove its manifest file from `/etc/kubernetes/manifests/` 
+and wait for 20 seconds (in case you have changed `fileCheckFrequency` field in [KubeletConfiguration struct](/docs/
+reference/config-api/kubelet-config.v1beta1/) you should wait as long as the new value set). 
+Then `kubelet` will kill the Pod and you can bring the manifest file back. 
+After another `fileCheckFrequency` period,  the `kubelet` will recreate the Pod and certificate renewal is complete.
```

Comment:
```suggestion
After running the command you should restart the control plane Pods. This is required since
dynamic certificate reload is currently not supported for all components and certificates.
[Static Pods](/docs/tasks/configure-pod-containerstatic-pod/) are managed by the local kubelet
and by the API Server, thus kubectl cannot be used to delete and restart them.
To restart a static Pod you can temporarily remove its manifest file from `/etc/kubernetes/manifests/` 
and wait for 20 seconds (see the `fileCheckFrequency` value in [KubeletConfiguration struct](/docs/
reference/config-api/kubelet-config.v1beta1/).
The kubelet will terminate the Pod and after that you can bring the manifest file back.
After another `fileCheckFrequency` period, the kubelet will recreate the Pod and the certificate renewal
for the component can complete.
```

note, you can click "commit" for my suggested change instead of opening a new PR after each requested change.

Reply:
Thanks for catching that. Updated.